docs/content_management/create_edit_pages.md

```diff
@@ -120,6 +178,20 @@ You cannot publish a Page if you have not set all the required settings of all b
 
 After you change the block settings, click **Submit** to save the changes or **Discard** to cancel.
 
+#### Save page
+
+There are several options for saving work on the page:
+
+|Option|Description|
+|------|-----------|
+|Close|Close the page without saving it.|
+|Send to review|Save the page and send it to review.|
+|Publish / Publish later|[Publish](publish_content.md) the page or schedule publishing for later. |
+|Save draft|Save the page draft*.|
+|Delete draft|Delete the page draft.|
```

Comment:
Publishing the page and deleting the draft actions are not related to saving the work. This is why I was using the convoluted sequence of steps in my original procedure.

Consider going back to the original procedure, renaming it to "Create a new page" and replacing the "Edit the page" step with "Edit page contents". It may start making sense then.

Reply:
It is related as the article walks a user through point A to point B.